types/react-plotly.js/index.d.ts

```diff
@@ -102,3 +102,5 @@ export interface PlotParams {
 
 export default class Plot extends React.PureComponent<PlotParams> {
 }
+
+export * from './factory';
```

Comment:
The package doesn't actually `export *` from `factory` (nor would that do anything since `export *` ignores default exports). I think the right thing to do is to add `factory.d.ts` to the files listed in `tsconfig.json`. Additionally, a test case would be appreciated.

Reply:
You are of course right.
I added the file to the tsconfig.json file and a test case to work with the minified version.